experimental/Experimental.jl

```diff
@@ -10,6 +10,7 @@ const oldexppkgs = [
   "ModStd",
   "Rings",
   "Schemes",
+  "FTheoryTools"
```

Comment:
```suggestion
  "FTheoryTools", # needs to be loaded after Schemes
```
adding a comment to not remember this for the future.

Reply:
Added to my last commit.